week3/runoff/runoff.c

```diff
@@ -0,0 +1,213 @@
+#include <cs50.h>
```

Comment:
This works and looks good, overall. 👍

You should remove existing comments and add speaking variables where possible.